modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/ActionButton/index.js

```diff
@@ -68,7 +68,7 @@ const ActionButton = ({
 		return (
 			<ButtonDropDown
 				items={activationKeysDownloadItems}
-				label={i18n.translate('download')}
+				label={i18n.translate('Download')}
```

Comment:
```suggestion
				label={i18n.translate('download')}
```

@luisacmn @claricebrasil, todas as chaves, de acordo com arquivo Language.json, sao minusculas. Provavalmente nao existe a palavra dentro do arquivo de tradução que precisa ser adicionado.